pkg/cmd/attestation/verify/verify.go

```diff
@@ -164,47 +167,115 @@ func runVerify(opts *Options) error {
 	if opts.PredicateType != "" {
 		filteredAttestations := verification.FilterAttestations(opts.PredicateType, attestations)
 
 		if len(filteredAttestations) == 0 {
-			return fmt.Errorf("no attestations found with predicate type: %s", opts.PredicateType)
+			opts.Logger.Printf(opts.Logger.ColorScheme.Red("✗ No attestations found with predicate type: %s\n"), opts.PredicateType)
+			return err
 		}
 
 		attestations = filteredAttestations
 	}
 
 	policy, err := buildVerifyPolicy(opts, *artifact)
 	if err != nil {
-		return fmt.Errorf("failed to build policy: %v", err)
+		opts.Logger.Println(opts.Logger.ColorScheme.Red("✗ Failed to build verification policy"))
+		return err
 	}
 
 	sigstoreRes := opts.SigstoreVerifier.Verify(attestations, policy)
 	if sigstoreRes.Error != nil {
-		return fmt.Errorf("at least one attestation failed to verify against Sigstore: %v", sigstoreRes.Error)
+		opts.Logger.Println(opts.Logger.ColorScheme.Red("✗ Verification failed"))
+		return sigstoreRes.Error
 	}
 
-	opts.Logger.VerbosePrint(opts.Logger.ColorScheme.Green(
-		"Successfully verified all attestations against Sigstore!\n",
-	))
-
-	opts.Logger.VerbosePrint(opts.Logger.ColorScheme.Green("Successfully verified the SLSA predicate type of all attestations!\n"))
-
-	opts.Logger.Println(opts.Logger.ColorScheme.Green("All attestations have been successfully verified!"))
+	opts.Logger.Println(opts.Logger.ColorScheme.Green("✓ Verification succeeded!\n"))
 
+	// If an exporter is provided with the --json flag, write the results to the terminal in JSON format
 	if opts.exporter != nil {
 		// print the results to the terminal as an array of JSON objects
 		if err = opts.exporter.Write(opts.Logger.IO, sigstoreRes.VerifyResults); err != nil {
-			return fmt.Errorf("failed to write JSON output")
+			opts.Logger.Println(opts.Logger.ColorScheme.Red("✗ Failed to write JSON output"))
+			return err
 		}
+		return nil
+	}
+
+	opts.Logger.Printf("%s was attested by:\n", artifact.DigestWithAlg())
+
+	// Otherwise print the results to the terminal in a table
+	tableContent, err := buildTableVerifyContent(sigstoreRes.VerifyResults)
+	if err != nil {
+		opts.Logger.Println(opts.Logger.ColorScheme.Red("failed to parse results"))
+		return err
+	}
+
+	headers := []string{"repo", "predicate_type", "workflow"}
+	if err = opts.Logger.PrintTable(headers, tableContent); err != nil {
+		opts.Logger.Println(opts.Logger.ColorScheme.Red("failed to print attestation details to table"))
+		return err
 	}
 
 	// All attestations passed verification and policy evaluation
 	return nil
 }
+
+func extractAttestationDetail(builderSignerURI string) (string, string, error) {
+	// If given a build signer URI like
+	// https://github.com/foo/bar/.github/workflows/release.yml@refs/heads/main
+	// We want to extract:
+	// * foo/bar
+	// * .github/workflows/release.yml@refs/heads/main
+	orgAndRepoRegexp := regexp.MustCompile(`https://github\.com/([^/]+/[^/]+)/`)
```

Comment:
fwiw this is already available from `Extensions.GithubWorkflowRepository` cc https://github.com/sigstore/sigstore-go/blob/c1df1ba603a289ca71de547e9ae3fb201be6ce87/pkg/verify/certificate_identity_test.go#L40